Basic-Video-Transformer/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "basic-video-transformer",
+  "version": "1.0.0",
+  "description": "This sample application shows how to use a basic video transformer",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "test"
+  },
+  "author": "Misha Behei, Christian Pettet",
```

Comment:
Nice 💪 
That said, might be malformed: https://docs.npmjs.com/cli/v8/configuring-npm/package-json#people-fields-author-contributors